nav2_route/graphs/geojson/aws_graph.geojson

```diff
@@ -0,0 +1,131 @@
+{
+"type": "FeatureCollection",
+"name": "nodes",
```

Comment:
I think we mentioned having a timestamp associated with the header of last edited?

Reply:
If that's possible!

Reply:
I added this to the aws_graph.geojson file.

Reply:
What about the timestamp? Not just the date

Reply:
Or is `urn:ogc:def:crs:EPSG::3857` some unique hash?

Reply:
I was following the same pattern as the lattice primitive files. I updated it to include the timestamp. 

That is the type of coordinate reference system so it is not a unique hash.